### PR TITLE
Restore solrj-plugin

### PR DESCRIPTION
**Pull request by dmsolr**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
#2871 
#2730

### CODE REVIEW

**Comment by dmsolr** [comment]
> ## Cases List
> 
> | Case description | Status | Cases|
> |:-----|:-----:|:-----:|
> |solrj| **12 passed. 0 failed**| [click me](#solrj) |
>

**Comment by wu-sheng** [comment]
@IanCao Could you re-run this, and we could merge this agian.

**Comment by dmsolr** [comment]
> inline

Sorry, @IanCao . I don't really understand what you mean.

**Comment by wu-sheng** [comment]
@IanCao I think you miss your comments.

**Comment by wu-sheng** [comment]
Here is the [test report](http://github.com/SkywalkingTest/agent-integration-test-report/tree/master/2019/6/dmsolr/testReport-master-2019-06-17-17-07.md) and [validate logs](http://35.227.88.23:49001/job/skywalking-agent-test/137/artifact/workspace/logs/validate_2019-06-17-17-07.log)

File: oap-server/server-core/src/test/resources/component-libraries.yml [hunk 1]
```diff
@@ -198,6 +198,12 @@ spring-cloud-gateway:
 RESTEasy:
   id: 62
   languages: Java
+SolrJ:
+  id: 63
+  languages: Java
+Solr:
```
Comment (kezhenxu94) [inline oap-server/server-core/src/test/resources/component-libraries.yml]: Is this what you will have in the other PR #2853 ? If so, leave it to where it should be

File: oap-server/server-starter/src/main/resources/component-libraries.yml [hunk 2]
```diff
@@ -216,6 +216,13 @@ spring-cloud-gateway:
 RESTEasy:
   id: 62
   languages: Java
+SolrJ:
+  id: 63
+  languages: Java
+Solr:
```
Comment (kezhenxu94) [inline oap-server/server-starter/src/main/resources/component-libraries.yml]: Is this what you will have in the other PR #2853 ? If so, leave it to where it should be

File: oap-server/server-core/src/test/resources/component-libraries.yml (hunk 1 above)
Comment (wu-sheng) [reply oap-server/server-core/src/test/resources/component-libraries.yml]: This is for both server plugin and server mapping. So it makes sense to be here from I known.

**Pull request merged**